src/Config.js

```diff
@@ -32,6 +32,7 @@ class ConfigOptions {
         this._stagingChecks = conf.staging_checks;
         this._loggerParams = conf.logger_params;
         this._approvalUrl = conf.approval_url;
+        this._coreDevelopers = conf.core_developers;
```

Comment:
Do we have to remember this "raw" value? If we can just use the derived this._coreDeveloperIds map, let's do that.